keyboards/massdrop/alt/keymaps/pregame/reading.md

```diff
@@ -0,0 +1,36 @@
+# Massdrop Alt Pregame
```

Comment:
This file can be completely deleted.

As I explained above, you may either work locally and push:

```
git rm -- keyboards/massdrop/alt/keymaps/pregame/reading.md
git commit -m "Delete reading.md"
git push
```

or [work remotely](https://docs.github.com/en/free-pro-team@latest/github/managing-files-in-a-repository/deleting-files) and pull:

```
[delete the file through GitHub.com, as detailed above]
[open your Git command-line interface, and run:]
git pull
```